samples/snippets/standard/async/async-and-await/vb/readme.md

```diff
@@ -21,7 +21,7 @@ The code for the *MainWindow.xaml.vb* file from this sample is included in the a
 
 ## Sample prerequisites
 
-This sample is written in Visual Basic and targets .NET Core 3.0 running on Windows. It requires the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+This sample is written in Visual Basic and targets .NET Core 3.0 running on Windows. It requires the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet/3.0).
```

Comment:
While you're at it, the sample is now using 3.1

https://github.com/dotnet/docs/blob/327f2c839b2cd78d3b22057963304ec06d9713c8/samples/snippets/standard/async/async-and-await/vb/AsyncFirstExampleVB.vbproj#L5